packages/model-viewer/src/test/three-components/Renderer-spec.ts

```diff
@@ -102,19 +102,6 @@ suite('Renderer', () => {
       expect(otherScene.renderCount).to.be.equal(1);
     });
 
-    test('marks scenes no longer dirty after rendering', () => {
```

Comment:
This test was flaky on Firefox 77, and its test case is already covered by the one immediately above (since we check that `otherScene` has not rendered a second time).

Reply:
i saw unit test fails at this test some times